data/sql/updates/pending_db_world/rev_1704609963729945200.sql

```diff
@@ -0,0 +1,17 @@
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 6268;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 6268);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(6268, 0, 0, 0, 0, 0, 100, 0, 5000, 11000, 11000, 17000, 0, 0, 11, 2691, 256, 0, 0, 0, 0, 5, 30, 0, 1, 0, 0, 0, 0, 0, 'Summoned Felhunter - In Combat - Cast \'Mana Burn\''),
+(6268, 0, 1, 0, 54, 0, 100, 512, 0, 0, 0, 0, 0, 0, 2, 35, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Summoned Felhunter - On Just Summoned -set faction 35'),
+(6268, 0, 2, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 80, 626800, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Summoned Felhunter - On Just Summoned - Run Script'),
```

Comment:
this seems broken, the comment says "On just summoned", but it's actually type "61 - LINK".
but this entry is not linked to anything? so it will never be executed.

When testing the felhunter actually stayed green / friendly and never engaged the fight.

Can you please re-check this? also please update the comments with Keira, thanks!